### ISSUE TITLE
Fatal error: `Unable to init PersistenceStore undefined`

### ISSUE THREAD

**Issue by mikenikles** [prisma/studio#900]
1. Prisma version (`prisma -v` or `npx prisma -v`):
```
prisma                  : 3.11.1
@prisma/client          : 3.9.2
Current platform        : debian-openssl-1.1.x
Query Engine (Node-API) : libquery-engine 1a2506facaf1a4727b7c26850735e88ec779dee9 (at ../../node_modules/.pnpm/@prisma+engines@3.11.1-1.1a2506facaf1a4727b7c26850735e88ec779dee9/node_modules/@prisma/engines/libquery_engine-debian-openssl-1.1.x.so.node)
Migration Engine        : migration-engine-cli 1a2506facaf1a4727b7c26850735e88ec779dee9 (at ../../node_modules/.pnpm/@prisma+engines@3.11.1-1.1a2506facaf1a4727b7c26850735e88ec779dee9/node_modules/@prisma/engines/migration-engine-debian-openssl-1.1.x)
Introspection Engine    : introspection-core 1a2506facaf1a4727b7c26850735e88ec779dee9 (at ../../node_modules/.pnpm/@prisma+engines@3.11.1-1.1a2506facaf1a4727b7c26850735e88ec779dee9/node_modules/@prisma/engines/introspection-engine-debian-openssl-1.1.x)
Format Binary           : prisma-fmt 1a2506facaf1a4727b7c26850735e88ec779dee9 (at ../../node_modules/.pnpm/@prisma+engines@3.11.1-1.1a2506facaf1a4727b7c26850735e88ec779dee9/node_modules/@prisma/engines/prisma-fmt-debian-openssl-1.1.x)
Default Engines Hash    : 1a2506facaf1a4727b7c26850735e88ec779dee9
Studio                  : 0.458.0
```

2. Logs from Developer Tools Console or Command line, if any:
<details>
  <summary>Screenshot</summary>
  <img src="https://user-images.githubusercontent.com/788827/161184320-c7512bdb-5b5f-440b-b97f-ed82979964e7.png"/>
</details>

3. Does the issue persist even after updating to the latest `prisma` CLI dev version? (`npm i -D prisma@dev`)
Yes

4. Prisma schema (if relevant):
N/A

## Additional notes

This started to happen to me as of today with Brave V1.37.109 (Mar 30, 2022).

Their [changelog](https://brave.com/latest/) mentions an upgrade to Chromium 100.0.4896.60 ([changelog](https://chromium.googlesource.com/chromium/src/+log/99.0.4844.88..100.0.4896.60?pretty=fuller&n=1000)).

The only reference to "indexed db" I found is a commit that reverts https://chromium-review.googlesource.com/c/chromium/src/+/3271759/.

Also worth mentioning, Studio works with Firefox.

**Comment by janpio** [comment]
Thanks for the report. Let's see if someone else has the same problem with Chrome(ium) 100.

**Comment by kleydon** [comment]
Consistently running into "Unable to init PersistenceStore" error (presented by dialog, in Studio), in Firefox.

**Comment by mikenikles** [comment]
This is no longer an issue for me with Chromium 101.0.4951.64. Given others reported the same issue with Firefox, I'm going to keep the issue open.

**Comment by athammer** [comment]
Can confirm having this issue with `Chrome: 106.0.5249.61` however not getting this on `Brave: Version 1.44.108 Chromium: 106.0.5249.103`

**Comment by kleydon** [comment]
Still an issue on FireFox.

**Comment by kleydon** [comment]
Additional context, from browser log on FireFox:
```
[PersistenceStore.init] Unable to init PersistenceStore undefined [index.js:1:1750](https://server-dev-jf52kv5fsq-uw.a.run.app/admin/prisma/assets/index.js)
DOMException: A mutation operation was attempted on a database that did not allow mutations. [index.js:1:1781](https://server-dev-jf52kv5fsq-uw.a.run.app/admin/prisma/assets/index.js)
Object { path: "PersistenceStore.init", code: undefined, type: undefined, message: "Unable to init PersistenceStore", stack: undefined, context: null, nativeError: DOMException }
[index.js:1:84080](https://server-dev-jf52kv5fsq-uw.a.run.app/admin/prisma/assets/index.js)
[BootstrapStore.init] Studio bootstrap failed 
Object { message: "Unable to init PersistenceStore", stack: undefined }
[index.js:1:1750](https://server-dev-jf52kv5fsq-uw.a.run.app/admin/prisma/assets/index.js)
Object { path: "PersistenceStore.init", code: undefined, type: undefined, message: "Unable to init PersistenceStore", stack: undefined, context: null, nativeError: DOMException }
[index.js:1:1781](https://server-dev-jf52kv5fsq-uw.a.run.app/admin/prisma/assets/index.js)
Uncaught (in promise) 
Object { path: "BootstrapStore.init", code: undefined, type: undefined, message: "Studio bootstrap failed", stack: undefined, context: {…}, nativeError: {…} }
```

**Comment by batmar125** [comment]
Yep! Same error on firefox. Using Cloudflare as a auth gateway for my database to keep it public. Y´all have a nice day!

**Comment by dchiasson** [comment]
I saw this same issue on Chrome (111.0.5563.65). The issue did not occur in incognito mode or using any other browser. I assume it was related to some state stored in the browser. I cleared all browser cache and cookies then restarted my browser to fix it.

**Comment by carolbenichio** [comment]
I had the same problem with Chrome. Works fine on Mozila.

**Comment by cloudbring** [comment]
I'm seeing the error on Chrome (versions / platform below). Oddly, Prisma Studio works in Incognito Mode

| Platform | Version |
|--------|------------------------|
| Chrome | Version 112.0.5615.137 (Official Build) (arm64) |
| MacOS | 13.3.1 (22E261) |

**Comment by hazlema** [comment]
I'm getting this error with Brave even with all plugins disabled and shields down.  However, It works using Google Chrome for me.

**Comment by hazlema** [comment]
I figured this out.. You need to allow ALL cookies fro localhost and 127.0.0.1.  For som reason just lowering the shields does not work
![image](https://github.com/prisma/studio/assets/203324/3972abeb-fc39-46bd-b2b4-726cfbcb8c11)

**Comment by igalklebanov** [comment]
Hey :wave:

Prisma ORM 7 introduces a brand new standalone [Studio](https://www.prisma.io/docs/orm/tools/prisma-studio), built from the ground up to work everywhere, with or without Prisma ORM. Consequently, all "old Studio" issues are being closed as there won't be any further work on it. :mega:

The new Studio is also available in the Prisma VS Code extension and in the Prisma Console application. :electric_plug:

Help shape, improve and fix the new Studio with us in this repository by submitting bug reports and feature requests. :construction_worker:
